web/sites/default/config/default/block.block.skillsdescription.yml

```diff
@@ -6,15 +6,14 @@ dependencies:
     - 'block_content:basic:b25cd33e-a4be-4ab8-b975-fc092df35dd5'
   module:
     - block_content
-    - domain
     - system
     - user
   theme:
     - nect
 id: skillsdescription
 theme: nect
-region: content
-weight: 8
+region: title_area
```

Comment:
@a-pasquale I already fixed the descriptions on this branch but will I need to do it on a new branch in order to export it?